packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -1844,6 +1848,14 @@
 	"settings.api.view.more-details": "You can find more details in",
 	"settings.api.view.external-docs": "the API documentation",
 	"settings.api.view.error": "Could not check if an api key already exists.",
+	"settings.api.view.modal.form.label": "Label",
+	"settings.api.view.modal.form.label.placeholder": "e.g Internal Project",
```

Comment:
Maybe rather something about credentials?

Reply:
credentials? 🤔

Reply:
Ooops sorry meant API keys :D Instead of projects

Reply:
I thought it was copy pasted from some project creation modal translation